tests/detect-uint-enum/test.rules

```diff
@@ -0,0 +1,2 @@
+alert websocket any any -> any any (msg:"There is no pong opcode in this packet"; websocket.opcode:!pong; sid:1;)
+alert websocket any any -> any any (msg:"There is no ping opcode in this packet"; websocket.opcode:!ping; sid:2;)
```

Comment:
I think you should just reuse the existing `websocket-ping` test, and add these rules to it